packages/react/src/checkbox/root/useCheckboxRoot.ts

```diff
@@ -80,7 +81,6 @@ export function useCheckboxRoot(params: UseCheckboxRoot.Parameters): UseCheckbox
     (externalProps = {}) =>
       mergeReactProps<'button'>(getValidationProps(externalProps), {
         ref: buttonRef,
-        value: 'off',
```

Comment:
I thought Switch ahd this at some point as well, but can't check the history easily

Reply:
It could make some sense if the value on the `button` was placed conditionally (only when unchecked) and also the `name` was set to the same value as the input. This way we could avoid rendering the hidden input conditionally, I suppose.